.travis.yml

```diff
@@ -0,0 +1,20 @@
+sudo: required
+languaje: node_js
+node_js:
+  - "9"
+services:
+  - docker
+before_install:
+  - wget -qO- https://toolbelt.heroku.com/install.sh | sh
+install:
+  - npm install
+script:
+  - docker build -t teams .
+deploy:
+  provider: script
+  skip_cleanup: true
```

Comment:
Travis also warns us that "skip_cleanup" is deprecated, we now have to use "cleanup" instead.